docs/docker/nginx_fpm/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.1'
+services:
+    db:
+        image: mysql:5.7
+        environment:
+            MYSQL_ROOT_PASSWORD: example
+            MYSQL_DATABASE: prestashop
+            MYSQL_USER: prestashop
+            MYSQL_PASSWORD: prestashop
+        networks:
+          - default
+
+    nginx:
+        image: nginx:1.14.0-alpine
+        ports:
+            - "8080:80"
+        volumes:
+            - ./prestashop/:/prestashop/
+            - ./prestashop-nginx/prestashop-nginx.conf:/etc/nginx/conf.d/default.conf
+        depends_on:
+          - "fpm"
+        networks:
+          - default
+
+    fpm:
+        build:
```

Comment:
You can reuse the image `prestashop/base:7.1-fpm` if you want to, it already contains some needed dependencies.

> https://github.com/PrestaShop/docker/blob/master/base/images/7.1-fpm/Dockerfile

Reply:
The image I use is a bit different alpine/nginx instead of debian/apache.
We have to work on the benchmark and see which variant we should push/support